### PR TITLE
Added remove_impediment stackeholder

### PR DESCRIPTION
**Pull request by amihaiemil**
PR for #1380 
Added the stackeholder + bundle test for it.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1441](https://github.com/zerocracy/farm/issues/1441) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1441](https://github.com/zerocracy/farm/issues/1441) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1991122488); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/groovy/com/zerocracy/stk/pm/in/impediments/remove_impediment.groovy [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import com.zerocracy.Project
+import com.zerocracy.farm.Assume
+import com.zerocracy.pm.ClaimIn
+import com.zerocracy.pm.in.Impediments
+
+/**
+ * This stackeholder responds to 'Remove impediment' event, which is
+ * triggered when the user says "@0crat continue" on an issue that is
+ * on hold.
+ * @todo #1380:30min The impediment should also be removed from the Agenda. We
```
Comment (krzyk) [inline src/main/groovy/com/zerocracy/stk/pm/in/impediments/remove_impediment.groovy]: @amihaiemil you need a longer todo :) (this one has only 17 words)

File: src/test/resources/com/zerocracy/bundles/remove_impediment/_after.groovy [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+package com.zerocracy.bundles.remove_impediment
+
+import com.jcabi.xml.XML
+import com.zerocracy.Project
+import com.zerocracy.pm.in.Impediments
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+
+def exec(Project project, XML xml) {
+    Impediments impediments = new Impediments(project).bootstrap().jobs()
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/remove_impediment/_after.groovy]: @amihaiemil `jobs()` returns `Iterable<String>` not `Impediments`

File: src/test/resources/com/zerocracy/bundles/remove_impediment/_before.groovy [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import com.zerocracy.Project
+import com.zerocracy.pmo.Awards
+import com.zerocracy.entry.ExtGithub
+
+def exec(Project project, XML xml) {
+    Farm farm = binding.variables.farm
+    Github github = new ExtGithub(farm).value()
+    Repo repo = github.repos().create(new Repos.RepoCreate('test', false))
+    repo.issues().create('On Hold', 'This issue has an impediment')
+    Impediments impediments = new Impediments(project).bootstrap().jobs()
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/remove_impediment/_before.groovy]: @amihaiemil return value of this method is incorrect

File: src/main/groovy/com/zerocracy/stk/pm/in/impediments/remove_impediment.groovy [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import com.zerocracy.Project
+import com.zerocracy.farm.Assume
+import com.zerocracy.pm.ClaimIn
+import com.zerocracy.pm.in.Impediments
+
+/**
+ * This stakeholder responds to 'Remove impediment' event, which is
+ * triggered when the user says "@0crat continue" on an issue that is
+ * on hold.
+ * @todo #1380:30min The impediment should also be removed from the Agenda. We
```
Comment (krzyk) [inline src/main/groovy/com/zerocracy/stk/pm/in/impediments/remove_impediment.groovy]: @amihaiemil why "removed from Agenda"? what will happen with such job?

Comment (amihaiemil) [reply src/main/groovy/com/zerocracy/stk/pm/in/impediments/remove_impediment.groovy]: @krzyk It is not the job that has to be removed from the agenda, but its impediment. Impediments are also specified in the agenda, apparently.

**Comment by amihaiemil** [comment]
@krzyk Right, I'll have a look later about that return type. I wrote that groovy code in notepad, basically...

File: src/main/groovy/com/zerocracy/stk/pm/in/impediments/remove_impediment.groovy (hunk 4 above)
Comment (krzyk) [reply src/main/groovy/com/zerocracy/stk/pm/in/impediments/remove_impediment.groovy]: @amihaiemil oh, ok

**Comment by amihaiemil** [comment]
@krzyk Fixed the problems now. Appveyor fails due to an unrelated problem, I think.

**Comment by krzyk** [comment]
@amihaiemil you missed one place: https://github.com/zerocracy/farm/pull/1441/files#r203402945

**Comment by krzyk** [comment]
@g4s8 ping

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1441](https://github.com/zerocracy/farm/issues/1441) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +31 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**